.changeset/modern-eels-judge.md

```diff
@@ -0,0 +1,7 @@
+---
+"@directus/app": minor
+"@directus/api": minor
+"tests-blackbox": minor
+---
+
+Added `Cache` option so that caching can be disabled for GET requests to Flows webhook trigger.
```

Comment:
The majority of the existing changesets don't end with a period:

```suggestion
Added `Cache` option so that caching can be disabled for GET requests to Flows webhook trigger
```